public/css/style.css

```diff
@@ -10,3 +10,8 @@
     background-color: var(--brand-color);
     border-color: var(--brand-color);
 }
+
+/* allows bootstrap to style Laminas Navigation generated menu items within the navbar */
+.navbar li.active a {
+    color: var(--bs-navbar-active-color);
+}
```

Comment:
I understand the motivation to solve the problem quickly for the tutorials, but it should not be here. :disappointed:
This means I'm fine with it for now but we need two new issue reports:

1. Update for the menu view helper is needed
2. Removing of this CSS rule

Reply:
1. An update to the view helper tutorial? Or to the view helper itself? If for the view helper itself then I would suggest adding the aria-current in the same change.
2. I'll create the removal issue so its not overlooked. I agree ideally it would not need to have this rule. But with the current way it is generated in the helper I thought it was the best solution for the moment.

Reply:
> …then I would suggest adding the aria-current in the same change.

Already available: https://github.com/laminas/laminas-view/releases/tag/2.27.0

Reply:
I created the issue as a feature request for the menu helper but i'm not real sure what category the other issue should fall under since its not actually in the code yet...

Reply:
@froschdesign 

I was wondering if there is a reason why the layout.phtml is missing:
```php
<base href="<?=$this->basePath();?>">
```
As you can imagine, it causes problems.

Also, I have no problem with working up the changes to the helper. I've just been really busy as of late. I am sure there will be a long list of test that needs to be updated if the class moves. I am also aware that there is related work going where the helpers will be moving etc. I just do not want to put a considerable amount of time into the changes if they are not going to be accepted :)

Reply:
> I was wondering if there is a reason why the layout.phtml is missing:
> 
> ```
> <base href="<?=$this->basePath();?>">
> ```
>
> As you can imagine, it causes problems.

What are the problems? The `base` element is only useful for _relative_ URLs in a document.